onboarding/src/Components/QnA/QnA.js

```diff
@@ -0,0 +1,144 @@
+import React, { useEffect, useState, useRef } from "react";
+import { v4 as uuidv4 } from "uuid";
+import MarkdownArea from "../MarkdownArea";
+import { deleteQnA } from "../hooks/QnAAPI";
+
+function QnA({
+  id,
+  question,
+  answer,
+  order,
+  qaList,
+  setQaList,
+  maxOrder,
+  setMaxOrder,
+  draggableProps,
+  innerRef,
+  dragHandleProps,
+  editMode,
+}) {
+  const [q, setQuestion] = useState(question);
+  const [a, setAnswer] = useState(answer);
+  const refValue = useRef(qaList);
+
+  useEffect(() => {
+    refValue.current = qaList;
+  });
+
+  const changeQuestion = (content) => {
+    setQuestion(content);
+    const updatedList = refValue.current.map((qa) => {
+      if (qa.id === id) {
+        qa.question = content;
+      }
+      return qa;
+    });
+    setQaList(updatedList);
+  };
+
+  const changeAnswer = (content) => {
+    setAnswer(content);
+    const updatedList = refValue.current.map((qa) => {
+      if (qa.id === id) {
+        qa.answer = content;
+      }
+      return qa;
+    });
+    setQaList(updatedList);
+  };
+
+  const handleCopyQA = (e) => {
+    e.preventDefault();
+    const updatedList = refValue.current.map((qa) => {
+      qa.order > order ? (qa.order = qa.order + 1) : qa;
```

Comment:
`_ESLint: Expected an assignment or function call and instead saw an expression.(no-unused-expressions)_`
I don't know why.